fuzz/fuzz-requirements.txt

```diff
@@ -1,3 +1,4 @@
 atheris
 atheris-libprotobuf-mutator
 protobuf
+json
```

Comment:
Is this needed?  Isn't json part of the python3 standard library set?

Reply:
I have python3 installed in my system but when i tried running the script locally it showed no module named json.